crates/walking_engine/src/parameters.rs

```diff
@@ -38,7 +38,7 @@ pub struct Base {
     pub foot_lift_apex_increase: Step,
     pub foot_offset_left: Vector3<Walk>,
     pub foot_offset_right: Vector3<Walk>,
-    pub step_duration: Duration,
+    pub step_duration: f32,
```

Comment:
I disagree with this change. This is an issue of deserializing values from the `default.json` but in code you always want to handle a `Duration`. `Duration` conveys units and has the proper functionality of adding/subtracting/etc.